addons/msg-count-badge/addon.json

```diff
@@ -8,32 +8,59 @@
   "versionAdded": "1.0.0",
   "settings": [
     {
-      "name": "Badge color",
-      "id": "color",
+      "name": "Badge color for message count",
+      "id": "countColor",
       "type": "color",
       "default": "#008000"
+    },
+    {
+      "name": "Indicate when not signed in",
+      "id": "showOffline",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "name": "Badge color when not signed in",
+      "id": "offlineColor",
+      "type": "color",
+      "default": "#e90404"
+    },
+    {
+      "name": "Badge indicator when not signed in",
+      "id": "offlineText",
+      "type": "string",
+      "default": "?",
+      "min": 1
```

Comment:
you can probably remove `min: 1` entirely

Reply:
I was originally thinking that leaving it blank would make the indicator disappear, which could be confusing to some users.

But then I realized that I could just manually add the space if the settings input is left blank.

But then I noticed that leaving the text field blank looks like this:

![image](https://user-images.githubusercontent.com/43426138/127591673-829218d1-1352-4b64-a419-29340471af99.png)

The `?` is the default, but it looks like it's still there, which could cause more confusion.

Reply:
try making `min: 0` then. i thought min was 1 by default.

Reply:
`min: 0` has the same effect.